### DISCUSSION TITLE
Animation requires Refresh

### DISCUSSION THREAD

**Discussion by tiagoml98** [mdbootstrap/TW-Elements#1966] [Support from community] [answered]
I'm struggling with what looks to be described in: https://stackoverflow.com/q/76678777/3183598
Upon importing an animation
e.g 
`import { Ripple, initTE } from 'tw-elements'

initTE({ Ripple })`

It works locally if I refresh the page or make a change to the source code, but I can't seem to get it to work when deployed.
I would appreciate any insight! Something tells me its a simple oversight on my end.

**Comment by juujisai** [comment, marked as answer]
Hi! Are you using react? Where do you initialize the Ripple component? Did you try doing it inside the `useEffect`?

```
  useEffect(() => {
    const init = async () => {
      const { Ripple, initTE } = await import("tw-elements");
      initTE({ Ripple}, { allowReinits: true });                - options if needed
    };
    init();
  }, []);
```

**Reply by tiagoml98** [reply]
I'm using Solid, but this worked really similarly. Thank you so much!
